lib/install.ps1

```diff
@@ -786,10 +796,21 @@ function find_dir_or_subdir($path, $dir) {
     return [string]::join(';', $fixed), $removed
 }
 
+function prepare_env_path ($path) {
```

Comment:
Would really love to not have to add this function unless it's absolutely necessary.